src/api/Alert/index.ts

```diff
@@ -391,6 +392,18 @@ export class AlertAPI extends Request {
     });
   }
 
+  /**
+   * Search for values of a given column in the cases of an alert.
+   */
+  public searchAlertColumnValues(this: Request<string[]>, params?: SearchAlertColumnValues) {
+    return this.request({
+      errors: [UNAUTHORIZED, DATA_SOURCE_UNAVAILABLE, FORBIDDEN, NOT_FOUND],
```

Comment:
any ideas for more errors @mzorkany ?

Reply:
@EricMallard maybe naming should camelCase ?